lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -253,7 +253,7 @@ DESC
         renderer = TaskRender.new
         total = tasks.count
 
-        tasks.each_with_index do |task, count|
+        tasks.find_each do |task, count|
```

Comment:
```
Gathering 106 tasks.
rake aborted!
NoMethodError: undefined method `+' for nil:NilClass
/home/aruzicka/git/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:258:in `block (4 levels) in <top (required)>'
/home/aruzicka/git/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:256:in `block (3 levels) in <top (required)>'
/home/aruzicka/git/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:250:in `block (2 levels) in <top (required)>'
/home/aruzicka/.rbenv/versions/2.3.1/bin/bundle:23:in `load'
/home/aruzicka/.rbenv/versions/2.3.1/bin/bundle:23:in `<main>'
Tasks: TOP => foreman_tasks:export_tasks
```

Apparently `find_each` yields just the items without the index. Would need something like `tasks.find_each.with_index do |task, count|`